src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java

```diff
@@ -0,0 +1,93 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.SentimentClassification;
+import com.codingnomads.betty.logic.models.SentimentResult;
+import edu.stanford.nlp.ling.CoreAnnotations;
+import edu.stanford.nlp.neural.rnn.RNNCoreAnnotations;
+import edu.stanford.nlp.pipeline.Annotation;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+import edu.stanford.nlp.sentiment.SentimentCoreAnnotations;
+import edu.stanford.nlp.trees.Tree;
+import edu.stanford.nlp.util.CoreMap;
+import org.apache.commons.lang3.StringUtils;
+import org.ejml.simple.SimpleMatrix;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+@Service
+public class SentimentAnalyserService {
+
+    @Autowired
+    @Qualifier("sentimentProperties")
+    private Properties properties;
+    @Autowired
+    private StanfordCoreNLP pipeline;
+
+    public List<SentimentResult> sentimentResultList = new ArrayList<>();
+
+    public SentimentResult getSentimentResult(String text) {
```

Comment:
This service should return a TeamSentimentScore which is the POJO the other service takes in. Not a SentimentResult (you could just delete this POJO completely)

Reply:
Same as my comment above.